src/Nethermind/Nethermind.AuRa.Test/AuRaBlockProducerTests.cs

```diff
@@ -121,7 +124,7 @@ public async Task Does_not_produce_block_when_ForceSealing_is_false_and_no_trans
             (await StartStop()).ShouldProduceBlocks(Quantity.None());
         }
         
-        [Test, Retry(10)] // TODO: fix the need for retry
+        [Test] // TODO: fix the need for retry
```

Comment:
was the need for retry fixed? then we can remove the comment

Reply:
Not sure if it was fixed, issue was long wait, works for now